k8s/business-create-ui.dc-srvc-rt-cmap-kcmap.param

```diff
@@ -4,32 +4,36 @@
 # Component: deployment config, service, route, config map, kc config map
 # Action: build
 # Template File: templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json
+# Hint: oc process -n mpgxod-dev -f templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json --param-file=business-create-ui.dc-srvc-rt-cmap-kcmap.param
+# Hint: oc process -n mpgxod-dev -f templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json --param-file=business-create-ui.dc-srvc-rt-cmap-kcmap.param | oc replace -n mpgxod-dev -f -
 #=========================================================
 NAME="business-create"
 DOCKER_REGISTRY="docker-registry.default.svc:5000"
 IMAGE_NAMESPACE="mpgxod-tools"
-TAG_NAME="test"
+TAG_NAME="dev"
```

Comment:
I didn't find this param file for Test or Prod. Are they needed?

Reply:
no we just need the one param file (at least that's how we're doing it in the backend). Less chance of making mistakes across configs that way and you have to fill in the secret values anyways

Reply:
What about all the environment-specific values specified below in this param file?

Reply:
cmd f 'dev' replace with 'test'/'prod' when you need is safer than having 3 param files that you need to keep in sync I think (all the other env specific ones are secrets). Already people forget to update just the 1 file. If even 1 out of 3 were out of sync it'd be a huge headache for ops on releases